payroll/views.py

```diff
@@ -103,12 +104,40 @@ def get_context_data(self, **kwargs):
         return context
 
     def median_entity_salary(self):
-        q = Salary.objects.filter(Q(job__position__employer__parent=self.object) | Q(job__position__employer=self.object))
+        q = Salary.objects.filter(
+            Q(job__position__employer__parent=self.object) |
+            Q(job__position__employer=self.object))
 
         results = q.all().aggregate(median=Percentile('amount', 0.5, output_field=FloatField()))
 
         return results['median']
 
+    def _make_pie_chart(self, container, entity_type, entity):
+        employer_payroll = entity_type.objects.filter(Q(id=entity.id)).aggregate(
+            extra_pay=Sum(Coalesce("departments__positions__jobs__salaries__extra_pay", 0)) +
+            Sum(Coalesce("positions__jobs__salaries__extra_pay", 0)),
+            base_pay=Sum(Coalesce("positions__jobs__salaries__amount", 0)))
```

Comment:
This is great! This comment has to do with formatting. In short, I'm a big fan of whitespace.

When I have multi-part operations, I find it a bit more readable to store them in variables, then pass them into the queryset, like:

```python
unit_base_pay = Sum(Coalesce('positions__jobs__salaries__amount, 0))
# etc. for extra pay and  department base / extra pay

models.objects.filter().aggregate(
    base_pay=unit_base_pay + dept_base_pay,
    extra_pay=...
)
```

This makes it a little easier to read and identify if you've left anything out.